### DISCUSSION TITLE
点击开始后504

### DISCUSSION THREAD

**Discussion by shuotianze** [Qexo/Qexo#453] [Q&A] [answered]
This Serverless Function has timed out.

Your connection is working correctly.

Vercel is working correctly.

504: GATEWAY_TIMEOUT
Code: FUNCTION_INVOCATION_TIMEOUT
ID: hnd1::b2r4p-1695048532498-8350166a9a96

这个是什么问题 大佬求解 数据库换了一个也不行

**Comment by am-abudu** [comment, marked as answer]
#392

**Reply by shuotianze** [reply]
谢谢
